.gitignore

```diff
@@ -17,3 +18,17 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+
+# Created by https://www.gitignore.io/api/vim
```

Comment:
You should probably just add this to your global gitignore. I don't think it belongs here.